test/e2e/basic_test.go

```diff
@@ -30,8 +31,12 @@ import (
 	appFramework "github.com/GoogleCloudPlatform/spark-on-k8s-operator/test/e2e/framework"
 )
 
+var namespace = os.Getenv("SPARK_TEST_NAMESPACE")
```

Comment:
We probably should keep things consistent by using flags instead of env variables.

Reply:
it is not easy to pass flags in there.

Reply:
@liyinan926 fixed that ^^